Yosemite/Yosemite/Stores/WooShippingStore.swift

```diff
@@ -163,6 +165,11 @@ private extension WooShippingStore {
                     completion: @escaping (Result<ShippingLabelPrintData, Error>) -> Void) {
         remote.printLabel(siteID: siteID, labelIDs: labelIDs, paperSize: paperSize, completion: completion)
     }
+
+    func loadOriginAddresses(siteID: Int64,
```

Comment:
Could we add unit tests for this?

Reply:
Sure, I forgot that there is a separate *Store and *Remote part 😅

Reply:
Done https://github.com/woocommerce/woocommerce-ios/pull/14689/commits/55c1bb5442968210ee70c01b9640423617c6b125#diff-cef15a0c0fca6dad2bde908883063d35c79fe66a99964880c867a79099a05dbf